Resources/templates/assets/css/modaal.css

```diff
@@ -0,0 +1,661 @@
+/*!
```

Comment:
En essayant ça, il me dit que ma branche est à jour...

Le 03/06/16 15:36, Godfraind a écrit :
>
> In Resources/templates/assets/css/modaal.css 
> <https://github.com/claroline/NetThemeBundle/pull/14#discussion_r65707699>:
>
> > @@ -0,0 +1,661 @@
> > +/*!
>
> tu dois faire |git add . -u| pour prendre en compte les suppressions ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/claroline/NetThemeBundle/pull/14/files/8e414f3cfc8362faa4fce96e3f80569601b9cfb5#r65707699>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ANpV2kr4Ak0Sk1mwecFxRpVrjxrJK3n0ks5qIC32gaJpZM4ItfhJ>.
>